pkg/services/sqlstore/migrations/dashboard_mig.go

```diff
@@ -187,10 +187,7 @@ func addDashboardMigration(mg *Migrator) {
 			{Name: "external_id", Type: DB_Text, Nullable: false},
 			{Name: "updated", Type: DB_DateTime, Nullable: false},
 		},
-		Indices: []*Index{
-			{Cols: []string{"dashboard_id"}},
-			{Cols: []string{"dashboard_id", "name"}, Type: IndexType},
-		},
+		Indices: []*Index{},
```

Comment:
Was this something left from another issue?

Reply:
Yupp. Could have been a separate PR but since it a prerequisite for this PR it makes sense to bundle them :)